titiler/pgstac/mosaic.py

```diff
@@ -191,6 +193,7 @@ def assets_for_bbox(
         TTLCache(maxsize=cache_config.maxsize, ttl=cache_config.ttl),
         key=lambda self, geom, **kwargs: hashkey(self.input, str(geom), **kwargs),
     )
+    @retry(tries=3, exceptions=pgErrors.OperationalError)
```

Comment:
In another implementation of a similar feature, we also retried on `InterfaceError`. Can't remember specific instances of that error, but seeing instances of errors like `psycopg2.InterfaceError: connection already closed` on the internets. Should this be included in the exception list?

Reply:
👍